anti-analysis/packer/gopacker/gopacker_capa.yml

```diff
@@ -0,0 +1,29 @@
+rule:
+  meta:
+    name: packed with GoPacker
+    namespace: anti-analysis/packer/gopacker
+    author: jared.wilson@fireeye.com
+    description: The sample appears to be packed with GoPacker.
+    scope: file
+    att&ck:
+    - Defense Evasion::Obfuscated Files or Information [T1027.002]
+    mbc:
+      - Anti-Static Analysis::Software Packing
+    examples:
+      - ebfba63891471ced641b60e0403777da
+      - 03ae008aa48e52d32517a60e2ce1a1d2
+    references:
+      - https://github.com/petercunha/gopacker
+  features:
+    - and:
+        - string: "Anonymous"
+        - string: "<autogenerated>"
+        - string: "github.com/nirhaas/gopacker/"
+        - string: ".packed"
+        - string: "ZSTDCompression"    
+        - string: "gopacker"
+        - string: "LALALALA"
+        - description: LALALALA is a unique and default magic footer for this packing. This footer can be changed, however, by default it is not.
+        - string: /\$f64\.[a-zA-Z0-9]{16}/
```

Comment:
regular expressions are a bit expensive to match (we have to scan *every* string for each regex). do you think this feature (and the associated `count: ...`) are strictly required to match this packer? if so - no problem! if not, then i'd recommend removing this feature as a performance tweak.

Reply:
Great question. I would think it's not. I just was trying to get as specific as possible to avoid FPs. However, let me remove it and run some tests to see if we can move forward without.